sdk/src/main/java/io/customer/sdk/util/Seconds.kt

```diff
@@ -9,8 +9,11 @@ data class Seconds(
     val value: Double
 ) {
     companion object {
-        // used for automated tests. Value that is small to make tests run fast, but long enough to test code's logic.
-        fun testValue(): Seconds = Seconds(0.01)
+        fun fromDays(numDays: Int): Seconds {
```

Comment:
We could utilize `TimeUnit` methods for these? 
`TimeUnit.DAYS.toSeconds()` and similarly 
`TimeUnit.SECONDS.toMillis()` for `toMilliseconds`

Reply:
Great suggestion. I forget that `TimeUnit` exists. 

I made the change:
```kotlin
data class Seconds(
    val value: Double
) {
    companion object {
        fun fromDays(numDays: Int): Seconds {
            return Seconds(TimeUnit.DAYS.toSeconds(numDays.toLong()).toDouble())
        }
    }

    val toMilliseconds: Milliseconds
        get() = Milliseconds(TimeUnit.SECONDS.toMillis(value.toLong()))

    override fun toString(): String = "$value seconds"
}
```

But the unit tests all failed. I think it's because we are converting `Double` (for seconds) into `Long` for `TimeUnit.ZZZ.toZZZ()` functions as they all require `Long`. `Double.toLong()` will remove the fractional part from `Double` so we are losing accuracy. 

So, we could need to manually convert `Double` to `Long` (probably by manually converting seconds into milliseconds) and then use `TimeUnit`. Which, at that point, we might as well skip using `TimeUnit`.